File: connect.pl
#!usr/bin/perl
# connect.pl: connect to the MySQL server

use strict;
use warnings;
use DBI;


my $dsn = "DBI:mysql:host=localhost;database=cookbook";
my $dbh = DBI->connect ($dsn, "sally", "sallypass")
	     or die "Cannot connect to server\n";

print "Connected\n";
$dbh->disconnect ();
print "Disconnected\n";

File: connect.py
#!/usr/bin/python
# connect.py: connect to the MySQL server

import mysql.connector

try:
    conn = mysql.connector.connect(database = "cookbook",
				   host = "localhost",
				   user = "sally",
				   password = "sallypass")
    print("Connected")
except:
    print("Cannot connect to server")
else:
    conn.close()
    print("Disconnected")


